controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -439,6 +436,9 @@ http {
             proxy_cookie_domain                     {{ $location.Proxy.CookieDomain }};
             proxy_cookie_path                       {{ $location.Proxy.CookiePath }};
 
+            # In case of errors try the next upstream server before returning an error
+            proxy_next_upstream                     {{ $location.Proxy.NextUpstream }}{{ if $cfg.RetryNonIdempotent }} non_idempotent{{ end }};
```

Comment:
Please create a function in the file template.go in order to avoid `non_idempotent` duplication (if the user added that option to the default)